docs/source/install.rst

```diff
@@ -0,0 +1,225 @@
+that will make sure all dependencies are installed, along with convenient development/programming tools.
+There are also video guides available for macOS (`here <https://youtu.be/0RGmmjed3Hc>`__), Raspberry Pi (`here <https://youtu.be/BpUMT-xqwqE>`__),
+Ubuntu (`here <https://youtu.be/QXeXMaxj4cM>`__), and Windows 10 (`here <https://youtu.be/ekopKJfcWiE>`__).
+
+macOS
+*****
+
+.. code-block:: bash
+
+  bash -c "$(curl -fL http://docs.luxonis.com/_static/install_dependencies.sh)"
+
+Close and re-open the terminal window after this command.
+
+The script also works on M1 Macs, Homebrew being installed under Rosetta 2, as some Python packages are still missing native M1 support.
+In case you already have Homebrew installed natively and things don't work, see `here <https://github.com/luxonis/depthai/issues/299#issuecomment-757110966>`__
+for some additional troubleshooting steps.
+
+Note that if the video streaming window does not appear consider running the
+following:
+
+.. code-block:: bash
+
+    python3 -m pip install opencv-python --force-reinstall --no-cache-dir
+
+See the `Video preview window fails to appear on macOS <https://discuss.luxonis.com/d/95-video-preview-window-fails-to-appear-on-macos>`_ thread on our forum for more information.
+
+Raspberry Pi OS
+***************
+
+.. code-block:: bash
+
+  sudo curl -fL http://docs.luxonis.com/_static/install_dependencies.sh | bash
+
+Ubuntu
+******
+
+.. code-block:: bash
+
+  sudo wget -qO- http://docs.luxonis.com/_static/install_dependencies.sh | bash
+
+Windows
+*******
+
+- Right click on Start
+- Choose Windows PowerShell (Admin)
+- Install Chocolatey package manager (similar to Homebrew for macOS):
+
+.. code-block:: bash
+
+  Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+
+- Close the PowerShell and then re-open another PowerShell (Admin) by repeating the first two steps.
+- Install Python and PyCharm
+
+.. code-block:: bash
+
+  choco install cmake git python pycharm-community -y
+
+Enabling the USB device (only on Linux)
+#######################################
+
+Since the DepthAI is a USB device, in order to communicate with it on the systems that use :code:`udev` tool, you
+need to add the udev rules in order to make the device accessible.
+
+The following command will add a new udev rule to your system
+
+.. code-block:: bash
+
+  echo 'SUBSYSTEM=="usb", ATTRS{idVendor}=="03e7", MODE="0666"' | sudo tee /etc/udev/rules.d/80-movidius.rules
+  sudo udevadm control --reload-rules && sudo udevadm trigger
+
+Install using pip
+#################
+
+Our packages are available to download using our `Artifactory server <https://artifacts.luxonis.com/artifactory/luxonis-python-snapshot-local/>`__.
```

Comment:
Instead of explanation first, lets do the installation first and we make that dynamic based on the current commit & version. Version can be retrieved with the script `find_version.py` and commit is part of git:
```
>>> import find_version
>>> print(find_version.get_package_version())
0.0.2.1
```
This makes the "latest" docs point to latest information by default without users need to discover

Reply:
I've added an issue to remember about updating the docs after release - https://github.com/luxonis/depthai-python/issues/91